great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -276,23 +277,28 @@ def add_csv_asset(
         Args:
             name: The name of the csv asset
             regex: regex pattern that matches csv filenames that is used to label the batches
+            glob_directive (str): glob for selecting files in directory (defaults to `**/*`) or nested directories (e.g. `*/*/*.csv`)
```

Comment:
I don't think you need `(str)` here, because we are using the annotations in docs.